konfera/models/event.py

```diff
@@ -4,33 +4,23 @@
 from konfera.models.abstract import FromToModel
 
 
-CONFERENCE = 'conference'
-MEETUP = 'meetup'
-
-EVENT_TYPE_CHOICES = (
-    (CONFERENCE, _('Conference')),
-    (MEETUP, _('Meetup')),
-)
-
-DRAFT = 'draft'
-PUBLISHED = 'published'
-PRIVATE = 'private'
-EXPIRED = 'expired'
-
-EVENT_STATUS_CHOICES = (
-    (DRAFT, _('Draft')),
-    (PUBLISHED, _('Published')),
-    (PRIVATE, _('Private')),
-    (EXPIRED, _('Expired')),
-)
-
-
 class EventManager(models.Manager):
     def published(self):
-        return self.get_queryset().filter(status=PUBLISHED)
+        return self.get_queryset().filter(status='published')
```

Comment:
Please do not compare to string, use constant instead (reason explained in Event model).

```
Event.PUBLISHED
```